schemas/JSON/configuration/configuration.schema.0.1.json

```diff
@@ -0,0 +1,204 @@
+{
+    "$id": "https://aka.ms/schemas/dsc/configuration.schema.0.1.yaml",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "A representation of a configuration used by an orchestrator for WinDSC.",
+    "type": "object",
+    "properties": {
+        "assertions": {
+            "type": "array",
+            "items": { "$ref": "#/defs/resource" }
+        },
+        "resources": {
+            "type": "array",
+            "items": { "$ref": "#/defs/resource" }
+        },
+        "parameters": {
+            "type": "array",
+            "items": { "$ref": "#/defs/resource" },
+            "description": "Resources that retrieve information via a 'get' operation."
+        },
+        "configurationType": {

```

Comment:
I don't understand what other values are expected here in the future.  Unless we very clearly have other schemas that will use other values and we use this to differentiate them, *and* those files are expected to be confusable with our own, we should hold off on adding it.

Additionally, `dsc` seems a bit too generic.

Reply:
Makes sense, removed "configurationType"